cypress/e2e/export-manager/search-filter-verify.cy.js

```diff
@@ -0,0 +1,122 @@
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Checkout from '../../support/fragments/checkout/checkout';
+import DateTools from '../../support/utils/dateTools';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import InteractorsTools from '../../support/utils/interactorsTools';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import FileManager from '../../support/utils/fileManager';
+
+let userData = {};
+const testData = {};
+const itemData = {
+  barcode: getRandomPostfix(),
+  instanceTitle: `Instance ${getRandomPostfix()}`,
+};
+const lastWeek = DateTools.getFormattedDate({ date: DateTools.getLastWeekDateObj() }, 'MM/DD/YYYY');
+const today = DateTools.getFormattedDate({ date: new Date() }, 'MM/DD/YYYY');
+const todayWithoutPadding = DateTools.getFormattedDateWithSlashes({ date: new Date() });
+const nextWeek = DateTools.getFormattedDate({ date: DateTools.getFutureWeekDateObj() }, 'MM/DD/YYYY');
+const exportRequestedCalloutMessage = 'Your Circulation log export has been requested. Please wait while the file is downloaded.';
+const jobCompletedCalloutMessage = 'Export job has been completed.';
+
+describe('export manager', () => {
+  before('create instance and user, check out item', () => {
+    cy.getAdminToken().then(() => {
+      cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => { testData.instanceTypeId = instanceTypes[0].id; });
+      cy.getHoldingTypes({ limit: 1 }).then((res) => { testData.holdingTypeId = res[0].id; });
+      cy.getLocations({ limit: 1 }).then((res) => { testData.locationId = res.id; });
+      cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanTypeId = res[0].id; });
+      cy.getMaterialTypes({ limit: 1 }).then((res) => { testData.materialTypeId = res.id; });
+      ServicePoints.getViaApi({ limit: 1 }).then((servicePoints) => { testData.servicepointId = servicePoints[0].id; });
+    }).then(() => {
+      InventoryInstances.createInstanceViaApi(itemData.instanceTitle, itemData.barcode);
+    });
+    cy.createTempUser([
+      permissions.exportManagerAll.gui,
+      permissions.circulationLogAll.gui,
+      permissions.checkoutAll.gui,
+    ])
+      .then(userProperties => {
+        userData = { ...userProperties };
+        UserEdit.addServicePointViaApi(testData.servicepointId,
+          userData.userId, testData.servicepointId);
+      })
+      .then(() => {
+        Checkout.checkoutItemViaApi({
+          id: uuid(),
+          itemBarcode: itemData.barcode,
+          loanDate: moment.utc().format(),
+          servicePointId: testData.servicepointId,
+          userBarcode: userData.barcode,
+        });
+
+        // Login and visit are separated, because otherwise user wasn't getting assigned permissions
+        cy.login(userData.username, userData.password);
```

Comment:
make it all in one line.
Like example:
    cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });

Reply:
please read comment on line 64